src/data-mgt/node/utils/transform.js

```diff
@@ -185,7 +185,10 @@ const transform = {
         (newObj, [field, value]) => {
           if (value) {
             let transformedField = "";
-            if (responseFromTransformFieldValues.DeviceType === "BAM") {
+            if (
+              responseFromTransformFieldValues.DeviceType === "BAM" ||
```

Comment:
just trying out case sensitivity checks. Not ideal solution. Just trial and error for now.